sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -177,14 +175,16 @@ public void testInsertIntoClosedPartition() throws Exception {
         execute("create table t (n integer) partitioned by (n)");
         ensureYellow();
```

Comment:
ensureYellow can be moved after the insert - the initial create table only creates a template and doesn't cause any shard allocations